README.md

```diff
@@ -56,17 +53,13 @@ By default, the pipeline currently performs the following:
 
```

Comment:
Ok so by reading the README im not exactly sure which variants it includes for example from COSMIC. Does it include them all?

Reply:
Must be quite a large DB in the end